runtime/compiler/infra/J9MonitorTable.cpp

```diff
@@ -51,6 +51,9 @@ J9::MonitorTable::init(
    table->_portLib = portLib;
    table->_monitors.setFirst(0);
 
+   table->_numCompThreads = 0;
+   table->_classUnloadMonitorHolders = NULL;
```

Comment:
Please add a comment saying that memory for classUnloadMonitorHolders will be allocated later, just before the compilation threads are started.